lib/containers/SendReceive.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { Header, Dropdown, Text, TabMenu } from '@bpanel/bpanel-ui';
+import { SendFunds, ReceiveFunds } from '../Components';
+
+import { sendReceive } from '../mappings';
+
+class SendReceive extends PureComponent {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    // fetch wallets
+    // set as state
+  }
+
+  render() {
+    const tabs = [
+      { header: 'Send Funds', body: <SendFunds /> },
+      { header: 'Receive Funds', body: <ReceiveFunds /> },
+    ];
+    return (
+      <div>
+        <Header type="h5">Select Wallet</Header>
+        <Dropdown options={[]} placeholder={'Select Wallet'} />
```

Comment:
when a prop is just a string, I believe you can just pass it without the curly braces: `placeholder="Select Wallet"` same as we do for classNames.